pkgs/applications/editors/emacs/elisp-packages/manual-packages/copilot/default.nix

```diff
@@ -3,10 +3,12 @@
   dash,
   editorconfig,
   fetchFromGitHub,
+  melpaBuild,
   nodejs,
   s,
-  melpaBuild,
```

Comment:
Not sure about moving parameters around.  If you really want to do this, please do so in a separate PR.